client/trino-cli/src/main/java/io/trino/cli/AlignedTablePrinter.java

```diff
@@ -130,6 +150,9 @@ public void printRows(List<List<?>> rows, boolean complete)
             int maxLines = 1;
             for (int i = 0; i < columns; i++) {
                 String s = formatValue(row.get(i));
+                if (decimalFields.get(i) && row.get(i) != null) {
+                    s = s + repeat(" ", maxDecimalWidth[i] - (s.length() - s.indexOf('.')));
```

Comment:
Please extract to a helpfully named function. Or add a comment.
It isn't very obvious to me at-least.